packages/daimo-common/src/foreignToken.ts

```diff
@@ -13,6 +35,15 @@ export type ForeignToken = {
   logoURI?: string;
 };
 
+export enum TokenLogo {
+  ETH = "https://assets.coingecko.com/coins/images/279/large/ethereum.png",
+  USDC = "https://assets.coingecko.com/coins/images/6319/large/usdc.png",
+  USDT = "https://assets.coingecko.com/coins/images/325/large/tether.png?1547034089",
+  DAI = "https://assets.coingecko.com/coins/images/9956/large/dai-multi-collateral-mcd.png?1574218774",
+}
```

Comment:
👍 

doesn't have to be in this PR but in future we can inline these as small base64 images